web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -147,7 +150,7 @@
             <xsl:attribute name="data-label" select="$label/label"/>
           </div>
           <div class="col-lg-2 gn-control">
-            <xsl:if test="not($isDisabled)">
+            <xsl:if test="not($isDisabled) and not($label/condition = 'mandatory')">
```

Comment:
Have you check if the element has cardinality N? Will not be possible to remove any of the items?